_posts/2022-11-29-graphdl/graphdl.Rmd

```diff
@@ -0,0 +1,547 @@
+---
+title: "Getting started with deep learning on graphs"
+description: >
+  This post introduces deep learning on graphs by mapping its central concept - message passing - to minimal usage patterns of PyTorch Geometric's foundation-laying MessagePassing class. Exploring those patterns, we gain some basic, very concrete insights into how graph DL works.
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: keydanagraphdl
+date: 2022-11-29
+categories:
+  - Torch
```

Comment:
Maybe use a Python tag, or `PyTorch`?

Reply:
Hm yeah ... for `Python`, it would be weird for some time since all others are R, and while we have that category for technical reasons, we don't display it ... [so far]

How about `Torch/PyTorch`? (structurally similar to e.g., `Packages/Releases`)

Reply:
I feel like in the Python world, 'Torch' (at least for experienced users) usually means the  Lua Torch, the PyTorch predecessor...

Yeah `Torch/Pytorch` seems fine. It would be nice to be able to list all Torch/PyTorch that use Python vs those that use R, though.

Reply:
Sounds good to me!

Reply:
doing PyTorch (standalone), for now (makes sense to wait before updating all Torch posts to the combined category)